### PR TITLE
Database/populate

### PR DESCRIPTION
**Pull request by m0uadh880**

### CODE REVIEW

File: src/test/python/entries.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+myCursor = mydb.cursor()
+
+
+def generate_consistent_entries(userid, start_date, subDuration):
+    date = start_date
+    end_date = start_date + timedelta(days=subDuration)
+
+    if subDuration == 0:
+        randomHour = random.randint(8, 20)
+        entry_time = datetime(date.year, date.month, date.day, randomHour, 0)
```
Comment (ounitech) [inline src/test/python/entries.py:23]: member can arrive at any time, so I think we can add minutes and not set them to 0.
it is a more realistic scenario

**Review by ounitech — CHANGES_REQUESTED** [review]

File: src/test/python/entries.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+myCursor = mydb.cursor()
+
+
+def generate_consistent_entries(userid, start_date, subDuration):
+    date = start_date
+    end_date = start_date + timedelta(days=subDuration)
+
+    if subDuration == 0:
+        randomHour = random.randint(8, 20)
```
Comment (ounitech) [inline src/test/python/entries.py:22]: let 's say gym is open for 6am to 10pm

File: src/test/python/entries.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        leave_time = datetime(date.year, date.month, date.day, randomHour + 2, 0)
+        myCursor.execute(insert_INTO_entry, (entry_time, leave_time, userid))
+        print(entry_time, leave_time, userid)
+        return
+
+    while date <= end_date:
+        if date.weekday() < 5:
+            randomHour = random.randint(8, 20)
+            entry_time = datetime(date.year, date.month, date.day, randomHour, 0)
+            leave_time = datetime(date.year, date.month, date.day, randomHour + 2, 0)
```
Comment (ounitech) [inline src/test/python/entries.py:33]: make it random  : members usually stay between 1h to 2hours. A member can stay 1h45minutes.

File: src/test/python/entries.py [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+    if subDuration == 0:
+        randomHour = random.randint(8, 20)
+        entry_time = datetime(date.year, date.month, date.day, randomHour, 0)
+        leave_time = datetime(date.year, date.month, date.day, randomHour + 2, 0)
+        myCursor.execute(insert_INTO_entry, (entry_time, leave_time, userid))
+        print(entry_time, leave_time, userid)
+        return
+
+    while date <= end_date:
+        if date.weekday() < 5:
```
Comment (ounitech) [inline src/test/python/entries.py:30]: gym is open all week . 
Members with regular job usually come 3 days in the week after work.
there is less people on week-end usually

File: src/test/python/entries.py (hunk 3 above)
Comment (m0uadh880) [reply src/test/python/entries.py:33]: Could I make the hours random so that a member might stay for 2 hours or more?

**Comment by m0uadh880** [comment]
script updated

**Review by ounitech — APPROVED** [review]

**Pull request merged**